sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -126,17 +112,15 @@ export class StudentFileMetadataAPIOutDTO {
   applicationNumber?: string;
 }
 
-export class StudentInfo {
+export class StudentProfileAPIOutDTO {
   firstName: string;
   lastName: string;
+  fullName: string;
   email: string;
   gender: string;
-  dateOfBirth: Date;
+  dateOfBirth: string;
```

Comment:
Any particular reason for changing date of birth to string ?

Reply:
This is a date-only value and there is no time associated with it. 
For these cases, mapping to a Date object, that has Date+Time, causes issues and is not accurate.
Always that we have a date-only we should map it to a string and have it parsed as below:
```
dateOfBirth: getISODateOnlyString(student.birthDate)
```